lms/static/sass/views/_homepage.scss

```diff
@@ -0,0 +1,168 @@
+// lms - views - homepage view
+// ====================
+
+$course-card-height: ($baseline*18);
+$course-image-height: ($baseline*8);
+$course-info-height: ($baseline*10);
+$course-title-height: ($baseline*3.6);
+$learn-more-horizontal-position: calc(50% - 100px); // calculate the left position for "LEARN MORE" content
+
+.courses-container {
+  @include outer-container;
+
+  .courses {
+    @include clearfix();
+    padding: ($baseline*2) 0 ($baseline*0.75) ;
+
+    .courses-listing {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .courses-listing-item {
+        @include fill-parent();
+        max-height: $course-card-height;
+        margin: ($baseline*0.75) 0;
+
+        @include media($medium) {
+          @include span-columns(4); // 4 of 8
+          @include omega(2n);
+        }
+
+        @include media($large) {
+          @include span-columns(4); // 4 of 12
+          @include omega(3n);
+        }
+
+        @include media($huge) {
+          @include span-columns(3); // 3 of 12
+          @include omega(4n);
+        }
+      }
+    }
+
+    .course {
+      @include box-sizing(border-box);
+      @include transition(all $tmg-f3 linear 0s);
+      box-shadow: 0 1px 10px 0 $black-t0, inset 0 0 0 1px $white-t3;
+      margin-bottom: ($baseline*1.5);
```

Comment:
Please move this margin to ``.courses-listing-item`` since it is part of the grid layout and many of the rules around a course in that layout are there.